src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Text } from "@artsy/palette"
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtworkAuctionCreateAlertHeader_artwork$data } from "__generated__/ArtworkAuctionCreateAlertHeader_artwork.graphql"
+import { ArtworkSidebarCreateAlertButtonFragmentContainer } from "Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCreateAlertButton"
+import { useFeatureFlag } from "System/useFeatureFlag"
+
+interface ArtworkAuctionCreateAlertHeaderProps {
+  artwork: ArtworkAuctionCreateAlertHeader_artwork$data
+}
+
+const ArtworkAuctionCreateAlertHeader: React.FC<ArtworkAuctionCreateAlertHeaderProps> = ({
+  artwork,
+}) => {
+  const getArtistNames = () => {
+    if (!artwork.artistNames) {
+      return "Artist Unavailable"
+    }
+    return artwork.artistNames
+  }
+
+  const isArtworkAuctionCreateAlertHeader = useFeatureFlag(
+    "onyx_auction-header-alert-cta"
+  )
+
+  if (!isArtworkAuctionCreateAlertHeader) {
+    return null
+  }
+
+  return (
+    <Box py={6}>
+      <Text variant="lg" textAlign={"center"}>
+        Bidding for <i>{artwork.title?.trim()}</i>, {getArtistNames()} has
```

Comment:
We could replace `getArtistNames()` function with just `artwork.artistNames ? artwork.artistNames : "Artist Unavailable"`

Reply:
We can also write it even shorter as `artwork.artistNames ?? "Artist Unavailable"` , updated 👍

Reply:
Hmmm not sure about that. Can `artwork.artistNames` be an empty array? If `artwork.artistNames` equals empty array, this line will return this empty array I believe

Reply:
I see there is a test to check if it's null but also an empty string, and in that case it would be empty. I am thinking now if we even want to show "Bidding for _Some Artwork_, Artist Unavailable has ended." maybe it makes sense to hide it overall. Checking now.

Reply:
(though I see we're using the same approach in Eigen for [My Collection](https://github.com/artsy/eigen/blob/ac43608fc6ea8457914cc143fae605fa8e654ce9/src/app/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkHeader.tsx#L54))

Reply:
In Eigen it's a bit different - this is the rule for personal artists - they will be shown only on the MyCollection pages

The auction result page is a bigger deal, I'd say. Nice catch, let's continue the discussion on Slack

Reply:
Ah no, I'm wrong, it's not only for personal artists

Reply:
As discussed, we are removing "Artist Unavailable" when artistNames are missing.